.github/workflows/checks.yml

```diff
@@ -30,7 +36,6 @@ jobs:
         env:
           CARGO_MAKE_RUN_CHECK_FORMAT: true
           CARGO_MAKE_RUN_CLIPPY: true
-          CARGO_MAKE_BUILD_EXAMPLES: true
```

Comment:
I removed this because it was trying to build examples without the `libquil` feature and thus failing on the libquil example. I added a `cargo build --examples --features libquil` elsewhere so they are still checked.

Reply:
Can the libquil example be marked `[cfg(feature="libquil")]`?

Reply:
I think the suggested thing is to mark it as requiring the `libquil` feature in `Cargo.toml` (see [here](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples) and [here](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#configuring-a-target)):

```toml
[[example]]
name = "libquil"
required-features = ["libquil"]
```

If I remember correctly how this works, it will just be skipped if the feature is not enabled.

Reply:
Just saw that you did this below. You should be able to re-enable this and not get failures.